sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerCertificateClient.cs

```diff
@@ -42,6 +42,8 @@ protected ConfidentialLedgerCertificateClient()
         /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
         /// <returns> The response returned from the service. Details of the response body schema are in the Remarks section below. </returns>
         /// <remarks>
+        /// Gets identity information for a Confidential Ledger instance.
```

Comment:
What's the value of adding this text that is already in the summary section? Do we have an example of where they are different?